docs/pywbemcli/commands.rst

```diff
@@ -1575,18 +1575,19 @@ environment).
 
 .. _Operation statistics:
 
-Statistics on WBEM server operations are maintained by the pywbemcli client,
-and also separately by some WBEM servers.  There are multiple components to
+Statistics on WBEM operations are maintained by the pywbemcli client, and also
+separately by WBEM servers that support this. There are multiple components to
 statistics gathering and reporting in pywbemcli:
 
-1.  Pywbemcli gathers and maintains statistics on CIM-XML operations it executes
-    against a WBEM server (or mock environment). These client statistics can be
-    displayed either after each pywbemcli command if the ``--timestats`` / ``-T``
-    general option is used, or in interactive mode with the ``statistics show``
-    command.
+1.  Pywbemcli gathers and maintains statistics on WBEM operations it executes
+    against a WBEM server (or mock environment). These client maintained
```

Comment:
how about changing these to the "The client maintained"

Reply:
DONE